cromshell

```diff
@@ -664,6 +664,25 @@ function submit()
 
   assertRequiredFileIsNonEmpty "${wdl}" "WDL"
   assertRequiredFileIsNonEmpty "${json}" "Input JSON"
+
+  # At this point, we should validate our inputs if we can:
+  which womtool &> /dev/null
+  r=$?
+  if [[ ${r} -eq 0 ]] ; then
+    echo -ne "Validating WDL and JSON before submission..."
+		local tmpValidationOutput=$( makeTemp )
+    womtool validate ${wdl} -i ${json} &> ${tmpValidationOutput}
+    if [[ $? -eq 0 ]] ; then
+      echo -e '\tSuccess!'
```

Comment:
This success might be confusing to people because it seems like the job was submitted here.  I would either change it to something like "WDL and JSON successfully validated` or just drop the message here.

Reply:
Yeah, I was on the fence.  I want to let the user know things validated, so I'll change the message.